python/pyspark/sql/column.py

```diff
@@ -151,6 +162,8 @@ def __init__(self, jc):
     __rdiv__ = _reverse_op("divide")
     __rtruediv__ = _reverse_op("divide")
     __rmod__ = _reverse_op("mod")
+    __pow__ = _bin_func_op("pow")
```

Comment:
I see, I'd like to go with current approach. 

We could change to `_bin_func_op` to `_pow`, use it for `__pow__` and `__rpow__`, it would be more clear.